lib/src/scanner.dart

```diff
@@ -160,18 +171,28 @@ class NgScanner {
       _state = NgScannerState.scanStart;
       return new NgToken.commentEnd(_current.offset);
     }
-    return handleError();
+    // Only triggered by EOF.
+    return handleError(
+      NgParserWarningCode.AFTER_COMMENT,
+      _lastCommentStartOffset,
+      _current.offset - _lastCommentStartOffset,
+    );
   }
 
+  // TODO: Max: Better handle cases like 'prop]'. Instead of
+  // TODO: 'prop []' resolve, resolve as '[prop].
```

Comment:
Yeah, I mean, its in a TODO anyway, mostly was just thinking out..loud, or...thinking...out...comment...